packages/clients/package.json

```
@@ -25,8 +33,15 @@
     "eslint-plugin-react-hooks": "^5.1.0-rc.0",
     "eslint-plugin-react-refresh": "^0.4.12",
     "globals": "^15.9.0",
-    "typescript": "^5.5.3",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.6.3",
     "typescript-eslint": "^8.7.0",
```

You'll have to update the version here.
```suggestion
    "typescript-eslint": "^8.11.0",
```